src/components/Charts/ModelChart.js

```diff
@@ -282,21 +233,56 @@ const ModelChart = ({
           },
         },
       },
-      series: projection.isInferred
-        ? [noAction, projected, shelterInPlace, availableBeds]
-        : [noAction, shelterInPlace, availableBeds],
```

Comment:
So I think maybe this needs to be:
```
      series: [noAction, projection.isInferred ? projected : shelterInPlace, availableBeds, previousEstimates]
```

Basically https://covid-projections-git-pablo-update-projections-chart.covidactnow.now.sh/us/wa/county/kittitas_county

should look a bit more like:
https://staging.covidactnow.org/us/wa/county/kittitas_county

Because we don't have inference results for many counties, so we have to show the static shelter-in-place (aka stay-at-home) scenario.  See https://blog.covidactnow.org/inference-projections-for-states/ for more details.

Reply:
Updated, see screenshot above